game/Doom/Game/p_setup.cpp

```diff
@@ -1646,6 +1648,36 @@ void P_SetupLevel(const int32_t mapNum, [[maybe_unused]] const skill_t skill) no
         ScriptingEngine::init();                        // PsyDoom: initialize the scripting engine if the map has Lua scripted actions
         MapHash::finalize();                            // PsyDoom: compute the final map hash
         MapPatcher::applyPatches();                     // PsyDoom: apply any patches to original map data that are relevant at this point, once all things have been loaded
+        // PsyDoom: if playing deathmatch or 'no monsters' setting is set, activate all special tagged boss sectors
```

Comment:
Minor:
Should be changed to say:
`// PsyDoom: if playing deathmatch and 'auto-activate special sectors' setting is enabled, activate all special tagged boss sectors`

Reply:
Thanks! The latest code has addressed this issue.